mantidproject/mantid
# Change from text-field to double spin-box in GetNegMuMuonicXRD

The change from manually editing the Y-Position shift seems like a lot of effort for the user, especially if they want to translate the plot while viewing it, so a double spin box has been implemented so the user can shift by simply incrementing their value.

**To Test** :
- Open Mantid and search for the algorithm `GetNegMuMuonicXRD`
- Observe that the `Y-position` field is a spinbox with the initial value '0.00'
- Select an `element`, `y-position` and enter a `OutputWorkspace` name.
- Run the algorithm and make sure no errors are produced.
- Make sure nothing but numerical values can be inserted into the SpinBox for Y-Positon as well.

Fixes #14348
